### PR TITLE
#16592 lowercase structure var name to remove content from index

### PR DESCRIPTION
**Pull request by erickgonzalez**

### CODE REVIEW

File: dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ContentletIndexAPIImpl.java [hunk 1]
```diff
@@ -930,12 +931,12 @@ public void removeContentFromLiveIndex(final Contentlet content) throws DotHiber
         removeContentFromIndex(content, true);
     }
 
-    public void removeContentFromIndexByStructureInode(String structureInode) throws DotDataException {
-        String structureName = CacheLocator.getContentTypeCache().getStructureByInode(structureInode).getVelocityVarName();
-        IndiciesInfo info = APILocator.getIndiciesAPI().loadIndicies();
+    public void removeContentFromIndexByStructureInode(final String structureInode) throws DotDataException {
+        final String structureName = CacheLocator.getContentTypeCache().getStructureByInode(structureInode).getVelocityVarName();
```
Comment (wezell) [inline dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ContentletIndexAPIImpl.java]: Deprecated - use ContentTypeAPI.find

Comment (wezell) [reply dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ContentletIndexAPIImpl.java]: also, this is open to a possible NPE

**Review by wezell — CHANGES_REQUESTED** [review]

**Comment by dsilvam** [comment]
Existing test was not failing. We need to fix it to it would have failed with the previous bug

File: dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ContentletIndexAPIImpl.java (hunk 1 above)
Comment (dsilvam) [reply dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ContentletIndexAPIImpl.java]: @erickgonzalez let's deprecate the two getter methods that are not deprecated on that class:
1. getStructureByInode
2. getStructureByVelocityVarName

**Review by jdotcms — APPROVED** [review]

File: dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ContentletIndexAPIImpl.java (hunk 1 above)
Comment (erickgonzalez) [reply dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ContentletIndexAPIImpl.java]: done

**Review by dsilvam — APPROVED** [review]

**Comment by wezell** [comment]
We need a timeout on the `DeleteByQueryAction`

**Pull request merged**